easybuild/easyblocks/l/lammps.py

```diff
@@ -480,7 +502,7 @@ def install_step(self):
 
             mkdir(site_packages, parents=True)
 
-            self.lammpsdir = os.path.join(self.builddir, '%s-*_%s' % (self.name.lower(), self.version))
+            self.lammpsdir = os.path.join(self.builddir, '%s-*%s' % (self.name.lower(), self.version))
```

Comment:
removing the version from the path: for a commit this will be the commit id instead
```suggestion
            self.lammpsdir = os.path.join(self.builddir, '%s-*' % self.name.lower())
```